backend/click

```diff
@@ -0,0 +1,285 @@
+#!/bin/bash
+# ==============================================================================
+# click
+#
+#   Thin client for interacting with running Urbit ship via conn.c.
+#
+#   Compiles a Hoon command given as text against an optional list of
+#   dependencies and then executes it on a running ship.
+#
+#   The click-format helper script handles command formatting, whereas this
+#   script handles operational control flow. See the documentation in
+#   click-format for the details of command structure, dependency management,
+#   etc.
+#
```

Comment:
Should probably leave a comment describing the changes this has compared to the "original" click, and why. ie timeout option and `socat`.